test/load/loadtests.py

```diff
@@ -268,3 +309,15 @@ def test_password_change_flow(self):
     def test_support_doc_flow(self):
         base_url = self.server_url[:-3]
         self.session.get(base_url + "/.well-known/browserid")
+
+    def _get_code_from_email(self, emailAcct, header):
+        # Due to possible race conditions, and previous test runs
+        # leaving junk in the restmail account, we look for the last
+        # matching email rather than the first.  There's still a small
+        # chance that we pull out the wrong code here.
+        for _ in xrange(10):
```

Comment:
Yeah, it tries to refetch the mailbox if it couldn't find a header. I guess the magic number made me go huh?